src/fetchers/repos-fetcher.js

```diff
@@ -0,0 +1,87 @@
+const { request } = require("../common/utils");
+const retryer = require("../common/retryer");
+
+const fetcher = (variables, token) => {
+    return request(
+        {
+            query: `
+      query getRepositories($login: String!,$top: Int=5) {
+        user(login: $login) {
+          repositories(first: $top, orderBy: {field: UPDATED_AT, direction: DESC}) {
+            totalCount
+            edges {
+                node {
+                  id
+                  name
+                  url
+                  updatedAt
+                  forkCount
+                  stargazers{
+                    totalCount
+                  }
+                }
+            }
+          }
+        }
+        organization(login: $login) {
+          repositories(first: $top, orderBy: {field: UPDATED_AT, direction: DESC}) {
+            totalCount
+            edges {
+                node {
+                  id
+                  name
+                  url
+                  updatedAt
+                  forkCount
+                  stargazers{
+                    totalCount
+                  }
```

Comment:
Why are you getting forkCount etc. if you dont use it?

Reply:
Hide border and title option is added and also not required fields for GraphQL request is removed for now.